helpers/whlsetup.sh

```diff
@@ -69,6 +69,7 @@ while [ $# -gt 0 ]; do
             shift
             ;;
```

Comment:
Please remove all the code block related to `-2` and just make `-3` the default.